underfs/hdfs/src/main/java/tachyon/underfs/hdfs/HdfsUnderFileSystem.java

```diff
@@ -121,14 +123,13 @@ public void close() throws IOException {
   @Override
   public FSDataOutputStream create(String path) throws IOException {
     IOException te = null;
-    int cnt = 0;
-    while (cnt < MAX_TRY) {
+    RetryPolicy retryPolicy = new CountingRetry(MAX_TRY);
+    while (retryPolicy.attemptRetry()) {
```

Comment:
What do you think of renaming `attemptRetry()` to just `attempt()`? The first attempt is counted as a "retry" the way it is now. What are your thoughts @calvinjia?